package.json

```diff
@@ -70,6 +70,7 @@
     "react-dom": "^16.1.0",
     "react-router": "^4.2.0",
     "react-router-dom": "^4.2.2",
-    "react-to-typescript-definitions": "^0.22.0"
+    "react-to-typescript-definitions": "^0.22.0",
+    "storybook-addon-material-ui": "^0.8.2"
```

Comment:
This PR is about type definition, not storybook. You shouldnot mix.

Storybook (and htis addon) are already scheduled, so useless to add this here or any PR.

Reply:
I just merged the master branch

Reply:
You should check https://github.com/Sharlaan/material-ui-superselectfield/pull/94/files

Reply:
Ah ok, probably Max started on storybook.